src/network/tarcap/packets_handlers/blocks_packet_handler.cpp

```diff
@@ -0,0 +1,62 @@
+#include "blocks_packet_handler.hpp"
+
+#include "dag/dag_block_manager.hpp"
+#include "network/tarcap/packets_handlers/common/syncing_handler.hpp"
+#include "network/tarcap/shared_states/syncing_state.hpp"
+
+namespace taraxa::network::tarcap {
+
+BlocksPacketHandler::BlocksPacketHandler(std::shared_ptr<PeersState> peers_state,
+                                         std::shared_ptr<PacketsStats> packets_stats,
+                                         std::shared_ptr<SyncingState> syncing_state,
+                                         std::shared_ptr<SyncingHandler> syncing_handler,
+                                         std::shared_ptr<DagBlockManager> dag_blk_mgr, const addr_t& node_addr)
+    : PacketHandler(std::move(peers_state), std::move(packets_stats), node_addr, "BLOCKS_PH"),
+      syncing_state_(std::move(syncing_state)),
+      syncing_handler_(std::move(syncing_handler)),
+      dag_blk_mgr_(std::move(dag_blk_mgr)) {}
+
+void BlocksPacketHandler::process(const dev::RLP& packet_rlp, const PacketData& packet_data,
+                                  const std::shared_ptr<dev::p2p::Host>& host __attribute__((unused)),
```

Comment:
`__attribute__((unused))` do not use this as we've already talked about it. It is not supported by all compilers. Maybe we can start using this https://en.cppreference.com/w/cpp/language/attributes/maybe_unused